l1-contracts/test/MultiProof.t.sol

```diff
@@ -90,6 +89,11 @@ contract MultiProofTest is RollupBase {
         )
       )
     );
+    // skip blob check if forge gas report is true
```

Comment:
This and the rollup.t.sol are both pulling from the rollup base. We could instead have a value that is written in the constructor or so to set it up neatly across all of them and not having multiple places needing to go look for env vars? Could also mean that this value is just set in that one place and not even needed in the others which seems nice

Reply:
An extra reason for this is that it would be inherited 👀 so we would avoid an issue as just encountered hehe where the ignition ones are using the blobs so it fails because not updated.